default.env

```diff
@@ -1,2 +1 @@
-LOG_LEVEL=DEBUG
-HEADER_KEYS_TO_PROXY=miauserid,miausergroups,miaclienttype,client-type,x-request-id
```

Comment:
@lucas-puerari we don't need any headers to be proxied?

Reply:
Nope, currently the ocr service is an end point service